### PR TITLE
General Survey on extra-graphics, extra-i18n (20210908)

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

Survey for updates in `extra-graphics/*` and `extra-i18n/*`, update where possible, rebuild otherwise.

Package(s) Affected
-------------------

See `groups/general-survey-20210908`.

Security Update?
----------------

No

Build Order
-----------

See `groups/general-survey-20210908`.

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-graphics/tachyon/autobuild/build [hunk 1]
```diff
@@ -1,5 +1,10 @@
 cd "$SRCDIR"/unix
 
+abinfo "Hacking to prohibit build-time stripping ..."
+ln -sv /usr/bin/true \
+    "$SRCDIR"/strip
```
Comment (KexyBiscuit) [inline extra-graphics/tachyon/autobuild/build:5]:
```suggestion
    /usr/local/strip
```

File: extra-graphics/tachyon/autobuild/build [hunk 2]
```diff
@@ -1,5 +1,10 @@
 cd "$SRCDIR"/unix
 
+abinfo "Hacking to prohibit build-time stripping ..."
+ln -sv /usr/bin/true \
```
Comment (KexyBiscuit) [inline extra-graphics/tachyon/autobuild/build:4]:
```suggestion
ln -fsv /usr/bin/true \
```

File: extra-graphics/tachyon/autobuild/build [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+    "$SRCDIR"/strip
+export PATH="$SRCDIR:$PATH"
```
Comment (KexyBiscuit) [inline extra-graphics/tachyon/autobuild/build:6]:
```suggestion
```

File: extra-graphics/tachyon/autobuild/patches/0001-Arch-Linux-tachyon-system-cflags.patch [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+ 	"ARCH = linux-64-thr" \
+ 	"CC = gcc" \
+-	"CFLAGS = -m64 -Wall -O3 -fomit-frame-pointer -ffast-math -DLinux -DLP64 -DTHR -D_REENTRANT $(MISCFLAGS)" \
++	"CFLAGS = $(CFLAGS) -DLinux -DLP64 -DTHR -D_REENTRANT $(MISCFLAGS)" \
+ 	"AR = ar" \
+ 	"ARFLAGS = r" \
+ 	"STRIP = strip" \
+ 	"RANLIB = ranlib" \
+-	"LIBS = -L. -ltachyon $(MISCLIB) -lm -lpthread"
++	"LIBS = -L. -ltachyon $(MISCLIB) -lm -lpthread" "LDFLAGS = $(LDFLAGS)"
```
Comment (KexyBiscuit) [inline extra-graphics/tachyon/autobuild/patches/0001-Arch-Linux-tachyon-system-cflags.patch:14]:
```suggestion
+	"LIBS = -L. -ltachyon $(MISCLIB) -lm -lpthread" \
+     "LDFLAGS = $(LDFLAGS)"
```

File: extra-i18n/fcitx5-pinyin-zhwiki/spec [hunk 5]
```diff
@@ -1,13 +1,13 @@
-dict_ver=20210722
+dict_ver=20210823
```
Comment (KexyBiscuit) [inline extra-i18n/fcitx5-pinyin-zhwiki/spec]: Use ALL_CAPITALIZED_ENV_VAR?

**Review by KexyBiscuit — APPROVED** [review]

**Comment by KexyBiscuit** [comment]
Looks fine overall, approved.

File: extra-graphics/tachyon/autobuild/patches/0001-Arch-Linux-tachyon-system-cflags.patch (hunk 4 above)
Comment (liushuyu) [reply extra-graphics/tachyon/autobuild/patches/0001-Arch-Linux-tachyon-system-cflags.patch:14]: You can't do this, this is a patch file.

**Pull request merged**